roles/icingadb_redis/defaults/main.yml

```
@@ -65,3 +65,4 @@ icingadb_redis_rdb_save_incremental_fsync: 'yes'
 #icingadb_redis_tls_cert: /etc/ssl/certs/host.crt
 #icingadb_redis_tls_key: /etc/ssl/private/host.key
 #icingadb_redis_tls_ca: /etc/ssl/certs/root-ca.crt
+#icingadb_redis_client_certificate:
```

Make the default `yes` as a string.

```suggestion
icingadb_redis_client_certificate: "yes"
```